parsers/nmparser/get_model_output.go

```diff
@@ -33,8 +34,14 @@ func GetModelOutput(filePath string, verbose bool, noExt bool, noGrd bool, noCov
 		if err != nil {
 			panic(err)
 		}
-		extData, _ := ParseExtData(ParseExtLines(extLines))
+		extData, parameterNames := ParseExtData(ParseExtLines(extLines))
 		results.ParametersData = extData
+
+		// check for Omega
+		if results.ParameterStructures.Omega == nil {
+			dim := getDimension(parameterNames.Omega[len(parameterNames.Omega)-1])
+			results.ParameterStructures.Omega = createDiagonalBlock(dim)
```

Comment:
this will not be the same parameterstrucutres type if it parses from the lst file as its doing a simple createDiagonalBlock

https://github.com/metrumresearchgroup/babylon/blob/test/nonmem/parsers/nmparser/parse_block_structure.go#L9:6

Look at the test results https://github.com/metrumresearchgroup/babylon/blob/test/nonmem/parsers/nmparser/parse_block_structure_test.go 

This currently would not do that

Reply:
The est file is only used when there is no data found in the lst file.  If we have block data we use it, if not we can create the diagonal based on the number of Omegas in the est.